conf/messages

```diff
@@ -2901,6 +2925,15 @@ reporting.frequency.table.mandated                                  = Quarterly
 reporting.frequency.table.voluntary                                 = Quarterly
 reporting.frequency.table.annual                                    = Annual
 
+#new table messages for reporting frequency with OptInOptOutContentUpdateR17 enabled:
+reporting.frequency.table.heading.usingMTD                          = Using Making Tax Digital for Income Tax
+reporting.frequency.table.heading.status.new                        = Your status
+reporting.frequency.table.MTD.voluntary                             = Yes
```

Comment:
I feel these keys should be .yes and .no

.MTD could be => .isUsingMtd or something along those lines